docs/getting_started.md

```diff
@@ -216,6 +219,94 @@ Here are some resources that can be used to get you started working with `AnnDat
 - [Converting directly from R to Python](https://theislab.github.io/scanpy-in-R/#converting-from-r-to-python)
 - [Homepage for scanpy tutorials](https://scanpy.readthedocs.io/en/latest/tutorials.html)
 
+
+## Special considerations for CITE-seq experiments
+
+
+If the dataset that you have downloaded contains samples with ADT data from a CITE-seq experiment, the raw and normalized ADT expression matrices are stored in the `altExp` of the `SingleCellExperiment` object in the `processed.rds` file.
+The following commands can be used to access the ADT expression matrices:
+
+```r
+# the raw ADT counts matrix
+raw_adt_counts <- counts(altExp(processed_sce))
+
+# the log normalized ADT counts matrix
+# if this value is NULL, it means normalization failed
+normalized_adt_counts <- logcounts(altExp(processed_sce))
+```
+
+It is important to be aware that the `SingleCellExperiment` object in the `processed.rds` file was not filtered based on ADT counts.
+The `adt_scpca_filter` column indicates which cells should be removed before proceeding with downstream analyses of the ADT data.
+Cells are labeled either as `"Keep"` (cells to retain) or `"Remove"` (cells to filter out).
+
+To filter cells based on this column, use the following command:
+```r
+# Filter cells based on ADT QC statistics
+processed_sce <- processed_sce[, which(processed_sce$adt_scpca_filter == "Keep")]
+```
+
+It is also important to be aware that the normalized ADT expression matrix only contains values for cells labeled as `"Keep"` in the `adt_scpca_filter` column.
+Any cells labeled `"Remove"` have `NA` values normalized expression matrix (see
+{ref}`processed adt data <processing_information:Processed ADT data>` for more details).
+
+As dimension reduction requires all known values (i.e., no `NA`s), filtering cells as shown above is critical before proceeding to downstream analyses on ADT counts.
+
+To perform principal components analysis (PCA) on the ADT counts, use the following `scater` command _after_ filtering has been performed:
+
+```r
+# Calculate PCA on the ADT counts
+processed_sce <- scater::runPCA(altExp(processed_sce))
+```
+
+To perform UMAP on this resulting PCA matrix, use the following `scater` command:
+
+```r
+# Calculate UMAP on the ADT counts
+processed_sce <- scater::runUMAP(altExp(processed_sce))
+```
```

Comment:
I don't know if we want to encourage people to do PCA & UMAP on the ADT counts alone. I don't think that this is a required part of a "getting started" doc, and I would probably just remove it.

Reply:
Ok, I might wait for a little discussion on this (maybe we can all chat next week!), since this was specifically added because of comments from Ally who is out right now. We'll regroup!

Reply:
I was only suggesting adding this to mirror the docs that we have for getting started with RNA. I've definitely seen PCA and UMAP on ADT's alone so don't think that's entirely crazy, but I think it's a question of how much we want to suggest. I think Josh has a point that we may not need to suggest it and maybe don't want to if that's something we haven't really had experience with ourselves. So I'm fine with removing it.